tests/shared/src/test/scala/munit/SuiteLocalFixtureSuite.scala

```diff
@@ -34,7 +34,7 @@ class SuiteLocalFixtureSuite extends FunSuite {
   }
 
   override def afterAll(): Unit = {
-    assertEquals(counter(), 17)
+    assertEquals(counter(), -10)
```

Comment:
This is also reverting to it's previous value:
https://github.com/scalameta/munit/pull/430/files#diff-2ead4e24b5b9e268cfb99fcec413a0d9c860af4663e31761a3f2822dab593f39